### ISSUE TITLE
Getting slightly worse performance than sklearn.model_selection.ShuffleSplit

### ISSUE THREAD

**Issue by StevenGubkin** [trent-b/iterative-stratification#30]
I have a detailed question, with code, here:

https://stats.stackexchange.com/questions/622880/multi-label-stratified-split

**Comment by trent-b** [comment]
Hi @StevenGubkin, I'm glad you were able to resolve the performance measure you were using and that you've found this package useful.
